content/1.2-rc/docs/installation/requirements.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Requirements"
+date: 2024-11-25
+---
+
+## Operators
+The orchestrator runtime/deployment is made of two main parts: `sonataflow operator` and `RHDH operator`  
+SonataFlow operator resource requirements are described [SonataFlow Installation Requirements](https://sonataflow.org/serverlessworkflow/latest/cloud/operator/install-serverless-operator.html#_prepare_a_minikube_instance)  
+The operator deploys a Data Index service and a Jobs service.
+These are the recommended minimum resource requirements for their pods:
+```
+Data Index pod:
+resources:
+      limits:
+        cpu: 500m
+        memory: 1Gi
+      requests:
+        cpu: 250m
+        memory: 64Mi
+```
+```
+Jobs pod:
+resources:
+      limits:
+        cpu: 200m
+        memory: 1Gi
+      requests:
+        cpu: 100m
+        memory: 1Gi
+```
+The resources for these pods are controlled by a CR of type SonataFlowPlatform. There is one such CR in the sonataflow-infra namespace.  
+The requirements for RHDH operator and its components are described [here](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/getting_started_with_red_hat_developer_hub/ref-rhdh-sizing_rhdh-getting-started#ref-rhdh-sizing_rhdh-getting-started)
+
+## Workflows
+Each workflow has its own logic and therefore different resource requirements that are influenced by its specific logic.  
+Here are some metrics for the workflows we provide. For each workflow you have the following fields: cpu idle, cpu peak (during execution), memory.
```

Comment:
Can you add the steps to collect those metrics, to assist user evaluate the resources used by their workflows?

Reply:
fixed